src/ui/Island/Modals/BetaEndModal/GetUpdatesForm.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FormEvent, useState } from "react"
+import Button from "shared/components/Interface/Button"
+
+export default function GetUpdatesForm() {
+  const [emailAddress, setEmailAddress] = useState("")
+
+  const formSubmitHandler = (e: FormEvent) => {
+    e.preventDefault()
+
+    // TODO: logic for sending form
```

Comment:
do we want to add a task to handle that later?

Reply:
Added here: https://github.com/tahowallet/dapp/issues/878